Cron/UpdateOrdersToProcessing.php

```diff
@@ -143,7 +147,15 @@ public function execute()
                 {
                     $rzpWebhookDataObj = unserialize($rzpWebhookData);
 
-                    $this->updateOrderStatus($order, $rzpWebhookDataObj);
+                    if (isset($rzpWebhookDataObj['payment.authorized']) === true)
+                    {
+                        $this->updateOrderStatus($order, 'payment.authorized', $rzpWebhookDataObj['payment.authorized']);
```

Comment:
@nikhilsharma1100 Please include value for `event` using constant variable
Example: `static::PAYMENT_AUTHORIZED`

Reply:
done